<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>LoginMode</class>
 <widget class="QWidget" name="LoginMode">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QPushButton" name="buttonNew">
     <property name="text">
      <string>New Game</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="buttonJoin">
     <property name="text">
      <string>Join Game</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="buttonQuit">
     <property name="text">
      <string>Quit Game</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
